README.md

```diff
@@ -427,6 +433,30 @@ const uiSchema = {
 };
 ```
 
+#### `addable` option
+
+if either `items` or `additionalItems` contains a schema object, an add button for new items is shown by default. You can turn this off with the `addable` option in `uiSchema`:
```

Comment:
This makes a lot of sense, I get the point. Also I realize that people may want to alter the UI according to some user ACLs, and your patch allows dealing with this in a very convenient fashion.

Scratch my concerns, this is valid.